fsspec/fuse.py

```diff
@@ -164,3 +213,122 @@ def run(fs, path, mount_point, foreground=True, threads=False):
             func()
         except KeyboardInterrupt:
             pass
+
+
+def mount(
+    url,
+    source_path,
+    mount_point,
+    log_file,
+    option,
+    foreground=True,
+    threads=False,
+    ready_file=False,
+):
+    kwargs = {}
+    for item in option:
+        key, value = item.split("=", 1)
+        if value.lower().endswith("[int]"):
+            value = int(value[: -len("[int]")])
+        elif value.lower().endswith("[bool]"):
+            value = value[: -len("[bool]")].lower() in ["1", "yes", "true"]
```

Comment:
Another alternative to this approach is just using `ast.literal_eval`. Instead of trying to manually cast types, you can get options in this form `a=1`, `a=True`, `a='xxx'` and then just call `ast.literal_eval(value)` which would just get you the actual python literal.

Reply:
In fact, I intentionally avoided forcing Python syntax on the command line. Also, instead of use `ast.literal_eval`, which involves more code to handle exceptions, I think it would be more simpler to check plain strings. But yes, I should add more document to explain that hard coded boolean values.

Reply:
Added explanatory information to docstring.